.github/workflows/main.yml

```diff
@@ -47,7 +47,9 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-
+    - name: Update npm
```

Comment:
I snagged this from the node-redis workflow: https://github.com/redis/node-redis/actions/runs/1583919219/workflow